src/__tests__/AllRaids.test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { loadingScreen, render, screen, userEvent } from '../tests/testUtils'
+import App from '../App'
+import { fetchedCollectionData } from '../tests/data/raidsData'
+
+beforeAll(() => {
+  jest.doMock('../utils/useCollection', () =>
+    jest.fn().mockReturnValue(fetchedCollectionData),
+  )
+})
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+it('should AllRaids links', async () => {
```

Comment:
I think this makes more sense in the integration test for now, especially since it is all subject to change